### PR TITLE
feat: Generate SudoMsg in contract

### PR DESCRIPTION
**Pull request by jawoznia**
#252

### CODE REVIEW

File: sylvia-derive/src/message.rs [hunk 1]
```diff
@@ -1290,32 +1290,7 @@ impl<'a> GlueMessage<'a> {
...
-            match (msg_ty, customs.has_msg) {
-                (MsgType::Exec, true) => quote! {
-                    #contract_enum_name:: #variant(msg) => #sylvia ::into_response::IntoResponse::into_response(msg.dispatch(contract, Into::into( #ctx ))?)
-                },
-                _ => quote! {
-                    #contract_enum_name :: #variant(msg) => msg.dispatch(contract, Into::into( #ctx ))
-                },
-            }
-        });
+        let dispatch_arms = interfaces.emit_dispatch_arms(msg_ty);
```
Comment (hashedone) [inline sylvia-derive/src/message.rs:1293]: Ideally, such refactoring should be one in separate PR. It could be accepted immediately, and this one would be simpler.

**Review by hashedone — COMMENTED** [review]
LGTM

File: sylvia-derive/src/message.rs (hunk 1 above)
Comment (jawoznia) [reply sylvia-derive/src/message.rs:1293]: you are right
